acceptance/tests/base/dsl/helpers/host_helpers/archive_file_from_test.rb

```diff
@@ -19,17 +19,20 @@
     assert_equal(true, Dir.exists?('archive/sut-files'))
   end
 
-  step "file is copied to local <archiveroot>/<host name> directory" do
+  step "archive is copied to local <archiveroot>/<host name> directory" do
     # Create a remote file to archive
     filepath = default.tmpfile('archive-file-test')
```

Comment:
I know this isn't part of your PR, but, do you know what happens if the `from_path` parameter is a directory and not a file? Or if it doesn't exist?